.github/scripts/backport.py

```diff
@@ -0,0 +1,461 @@
+            comments = get_issue_comments(number)
+            prev_time = last_reminder_time(comments, COMMENT_MARKER_BASE)
+            if prev_time is None:
+                post_comment(number, f"{COMMENT_MARKER_BASE}\n@{author}\n{REMINDER_BODY}")
+                print(f"[reminder] PR #{number}: initial reminder posted")
+            elif prev_time < threshold:
+                post_comment(number, f"{COMMENT_MARKER_BASE}\n@{author}\n{REMINDER_BODY}")
+                print(f"[reminder] PR #{number}: follow-up reminder posted (prev {prev_time.isoformat()})")
+            else:
+                print(f"[reminder] PR #{number}: cooling period not elapsed (prev {prev_time.isoformat()})")
+        except Exception as ex:
+            print(f"::error:: PR #{pr.get('number', '?')}: {ex}", file=sys.stderr)
+            continue
+    return 0
+
+
+# ----------------------------- CLI -----------------------------
+def parse_args(argv: List[str]) -> argparse.Namespace:
+    try:
+        parser = argparse.ArgumentParser(description="Backport utilities")
+        parser.add_argument(
+            "--repo",
+            help="Target repository in owner/repo form (overrides GITHUB_REPOSITORY env)",
+            required=False,
+        )
+        parser.add_argument(
+            "--trigger",
+            choices=["pull_request_target", "schedule", "workflow_dispatch"],
+            required=True,
+            help="Mode of operation",
+        )
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            help="Simulate actions without modifying GitHub state",
+        )
+        parser.add_argument(
+            "--debug",
+            action="store_true",
+            help="Enable verbose debug logging",
+        )
+        sub = parser.add_subparsers(dest="command", required=True)
+
+        p_label = sub.add_parser("label", help="Add backport pending label to merged PRs without version label")
+        p_label.add_argument(
+            "--lookback-days",
+            type=int,
+            required=False,
+            default=1,
+            help="Days to look back when not pull_request_target (default: 1). Ignored for pull_request_target single PR",
+        )
+        p_label.add_argument(
+            "--remove",
+            action="store_true",
+            required=False,
+            default=False,
+            help="Removes backport pending label",
+        )
+
+        p_remind = sub.add_parser("remind", help="Post reminders on merged PRs still pending backport")
+        p_remind.add_argument(
+            "--lookback-days",
+            type=int,
+            required=True,
+            help="Days to look back for updated merged PRs",
+        )
+        p_remind.add_argument(
+            "--pending-label-age-days",
+            type=int,
+            required=True,
+            help="Days between reminders for the same PR",
+        )
```

Comment:
Is there a strong reason to keep those required instead of providing reasonable defaults?